README.md

```diff
@@ -50,3 +50,16 @@ The name of the index types isn't always obvious to the public so on the site th
 * Objects => Object
 * Documents => Archive
 * Location => Facility
+
+### Routes
+
+To access the pages we currently have templates for, use:
+
+* `/`
+* `/search?q={query}`
+* `/object/{id}`
+* `/person/{id}`
+* `/archive/{id}`
```

Comment:
Could you fix these up so they reflect the API docs: https://github.com/TheScienceMuseum/collectionsonline/wiki/Collections-Online-API#get-objectsidslug